subsys/usb/class/dfu/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_sources(usb_dfu.c)
+
+if(CONFIG_USB_DEVICE_VID EQUAL CONFIG_USB_DEVICE_DFU_VID)
```

Comment:
Please remove everything about USB_DEVICE_DFU_VID, it is not necessary to change VID.

Reply:
Done